packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -539,8 +539,9 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
     HapticFeedback.selectionClick();
     widget.focusNode.requestFocus();
     _selectWordAt(offset: details.globalPosition);
-    _showToolbar();
-    _showHandles();
+    if (defaultTargetPlatform != TargetPlatform.android) {
```

Comment:
Can you add a comment here mentioned other platforms show handles when long press ends